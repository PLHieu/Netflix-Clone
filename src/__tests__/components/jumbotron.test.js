import { render } from '@testing-library/react';
import React from 'react';
import JumbotronContainer from '../../containers/jumbotron.container';

describe('<Jumbotron/>', () => {
  it('render the jumbotron with populated data', () => {
    const { container, getByText, getByAltText } = render(<JumbotronContainer />);

    expect(getByText('Enjoy on your TV.')).toBeTruthy();
    expect(getByAltText('Tiger King on Netflix')).toBeTruthy();
    expect(
      getByText('Watch on smart TVs, PlayStation, Xbox, Chromecast, Apple TV, Blu-ray players and more.')
    ).toBeTruthy();

    expect(container.firstChild).toMatchSnapshot();
  });
});
