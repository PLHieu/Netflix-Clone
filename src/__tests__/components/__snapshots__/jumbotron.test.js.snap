// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Jumbotron/> render the jumbotron with populated data 1`] = `
<div
  class="jumbotron__Container-sc-zerd7v-6 cuWyCS"
>
  <div
    class="jumbotron__Item-sc-zerd7v-5 eZQsWr"
  >
    <div
      class="jumbotron__Inner-sc-zerd7v-0 etFpoe"
      direction="row"
    >
      <div
        class="jumbotron__Pane-sc-zerd7v-1 iIujeb"
      >
        <h1
          class="jumbotron__Title-sc-zerd7v-2 ejeskW"
        >
          Enjoy on your TV.
        </h1>
        <h2
          class="jumbotron__SubTitle-sc-zerd7v-3 kCFOvl"
        >
          Watch on smart TVs, PlayStation, Xbox, Chromecast, Apple TV, Blu-ray players and more.
        </h2>
      </div>
      <div
        class="jumbotron__Pane-sc-zerd7v-1 iIujeb"
      >
        <img
          alt="Tiger King on Netflix"
          class="jumbotron__Image-sc-zerd7v-4 ePZUUV"
          src="/images/misc/home-tv.jpg"
        />
      </div>
    </div>
  </div>
  <div
    class="jumbotron__Item-sc-zerd7v-5 eZQsWr"
  >
    <div
      class="jumbotron__Inner-sc-zerd7v-0 mzURf"
      direction="row-reverse"
    >
      <div
        class="jumbotron__Pane-sc-zerd7v-1 iIujeb"
      >
        <h1
          class="jumbotron__Title-sc-zerd7v-2 ejeskW"
        >
          Download your programmes to watch on the go.
        </h1>
        <h2
          class="jumbotron__SubTitle-sc-zerd7v-3 kCFOvl"
        >
          Save your data and watch all your favourites offline.
        </h2>
      </div>
      <div
        class="jumbotron__Pane-sc-zerd7v-1 iIujeb"
      >
        <img
          alt="Watch on mobile"
          class="jumbotron__Image-sc-zerd7v-4 ePZUUV"
          src="/images/misc/home-mobile.jpg"
        />
      </div>
    </div>
  </div>
  <div
    class="jumbotron__Item-sc-zerd7v-5 eZQsWr"
  >
    <div
      class="jumbotron__Inner-sc-zerd7v-0 etFpoe"
      direction="row"
    >
      <div
        class="jumbotron__Pane-sc-zerd7v-1 iIujeb"
      >
        <h1
          class="jumbotron__Title-sc-zerd7v-2 ejeskW"
        >
          Watch everywhere.
        </h1>
        <h2
          class="jumbotron__SubTitle-sc-zerd7v-3 kCFOvl"
        >
          Stream unlimited films and TV programmes on your phone, tablet, laptop and TV without paying more.
        </h2>
      </div>
      <div
        class="jumbotron__Pane-sc-zerd7v-1 iIujeb"
      >
        <img
          alt="Money Heist on Netflix"
          class="jumbotron__Image-sc-zerd7v-4 ePZUUV"
          src="/images/misc/home-imac.jpg"
        />
      </div>
    </div>
  </div>
</div>
`;
